<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Houdini Template Publisher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2e3138;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="bundle_type">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    padding: 5px;
    padding-left: 10px;
    color: #333333;
    min-width: 6em;
	font-size: 17px;
}

QComboBox:hover {
    background-color: #d0d0d0;
    border-color: #909090;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;

}

QComboBox::down-arrow {
    image: url(&quot;D:/houdini_template_publisher/icons/arrow-down-3101.svg&quot;);
    width: 2px;
    height: 2px;
}

QComboBox QAbstractItemView {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    selection-background-color: #c0c0c0;
    color: #333333;
    outline: none;
}

QComboBox QAbstractItemView::item {
    padding: 5px;
    padding-left: 10px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #d0d0d0;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #c0c0c0;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>Select Bundle</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="bundle_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    color: black;
    font-family: &quot;Asap&quot;;
	font-size: 17px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Type Bundle Name ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string> Bundle Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_template">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
	color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>Template</string>
    </property>
   </widget>
   <widget class="QComboBox" name="category">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    padding: 5px;
    padding-left: 10px;
    color: #333333;
    min-width: 6em;
	font-size: 17px;
}

QComboBox:hover {
    background-color: #d0d0d0;
    border-color: #909090;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;

}

QComboBox::down-arrow {
    image: url(&quot;D:/houdini_template_publisher/icons/arrow-down-3101.svg&quot;);
    width: 2px;
    height: 2px;
}

QComboBox QAbstractItemView {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    selection-background-color: #c0c0c0;
    color: #333333;
    outline: none;
}

QComboBox QAbstractItemView::item {
    padding: 5px;
    padding-left: 10px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #d0d0d0;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #c0c0c0;
}</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="is_nodesnippet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
	color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size:17px;
}</string>
    </property>
    <property name="text">
     <string>Node Snippet</string>
    </property>
   </widget>
   <widget class="QComboBox" name="context">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    padding: 5px;
    padding-left: 10px;
    color: #333333;
    min-width: 6em;
	font-size: 17px;
}

QComboBox:hover {
    background-color: #d0d0d0;
    border-color: #909090;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;

}

QComboBox::down-arrow {
    image: url(&quot;D:/houdini_template_publisher/icons/arrow-down-3101.svg&quot;);
    width: 2px;
    height: 2px;
}

QComboBox QAbstractItemView {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    selection-background-color: #c0c0c0;
    color: #333333;
    outline: none;
}

QComboBox QAbstractItemView::item {
    padding: 5px;
    padding-left: 10px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #d0d0d0;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #c0c0c0;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="image_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    color: black;
    font-family: &quot;Asap&quot;;
	font-size: 17px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>Image Sequence Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>Add Tags</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tags">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    padding: 5px;
    padding-left: 10px;
    color: #333333;
    min-width: 6em;
	font-size: 17px;
}

QComboBox:hover {
    background-color: #d0d0d0;
    border-color: #909090;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;

}

QComboBox::down-arrow {
    image: url(&quot;D:/houdini_template_publisher/icons/arrow-down-3101.svg&quot;);
    width: 2px;
    height: 2px;
}

QComboBox QAbstractItemView {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    selection-background-color: #c0c0c0;
    color: #333333;
    outline: none;
}

QComboBox QAbstractItemView::item {
    padding: 5px;
    padding-left: 10px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #d0d0d0;
}

QComboBox QAbstractItemView::item:selected {
    background-color: #c0c0c0;
}</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="tags_scroll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
	background-color: red;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>479</width>
       <height>129</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="publish">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>730</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius: 5px;
    width: 100px;
    height: 36px;
    background-color: #8ab71b;
    color: #fff;
    margin: 8px;
    font-family: &quot;Roboto&quot;;
    font: bold;
    font-size: 20px;
}

QPushButton:hover{
	background-color: #649100;
}


QPushButton:pressed {
   background-color: #047bad;
}</string>
    </property>
    <property name="text">
     <string>Publish</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #ff6600;
    font-family: &quot;Consolas&quot;;
    font: bold;
    font-size: 25px;
	background: linear-gradient(to right, #ff6600, #ff9a44);
	font-weight: 700
}</string>
    </property>
    <property name="text">
     <string>Template Publisher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>Enter Description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: #e0e0e0;
    border: 1px solid #a0a0a0;
    border-radius: 3px;
    color: black;
    font-family: &quot;Asap&quot;;
	font-size: 17px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Descriptions....</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bundle_icon">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="category_icon">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="context_icon">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tag_icon">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="snapshot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            background-color: rgb(0, 20, 84);  /* Google blue */
            color: white;
            border: none;
            border-radius: 50%;  /* Circular button */
            
            font-size: 17px;
            font-weight: bold;
        }
        
        QPushButton:hover {
            background-color:  #2e3138;  /* Slightly darker blue */
        }
        
        QPushButton:pressed {
            background-color: #2A56C6;  /* Even darker blue */
        }</string>
    </property>
    <property name="text">
     <string>+ Add Snapshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_dialog">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
            background-color:rgb(127, 134, 150);  /* Google blue */
            color: white;
            border: none;
            border-radius: 50%;  /* Circular button */
            
            font-size: 17px;
            font-weight: bold;
        }
        
        QPushButton:hover {
            background-color:  rgb(90, 112, 89);  /* Slightly darker blue */
        }
        
        QPushButton:pressed {
            background-color: #2A56C6;  /* Even darker blue */
        }</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="latest_version">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #fff;
    font-family: &quot;Asap&quot;;
    font: bold;
    font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blink_text">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
            color: rgb(255, 162, 0);
            border: none;
            border-radius: 50%;  /* Circular button */
            
            font-size: 18px;
            font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>[Select Nodes To Publish]</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
